aafactory/src/tests/test_chat/test_mock_chat.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from aafactory.chat.interface import _load_avatar_infos_for_chat, send_request_to_llm, CHAT_HISTORY
+
+@pytest.mark.asyncio
+async def test_chat_with_avatar_mock(mocker):
+    # Clear global state
+    CHAT_HISTORY.clear()
+    
+    # Mock the common dependencies (open ai, comfyui)
+    mocker.patch("aafactory.chat.interface.send_request_to_open_ai", new_callable=mocker.AsyncMock, return_value="Hi there!")
+    mocker.patch("aafactory.chat.interface.send_request_to_generate_video", new_callable=mocker.AsyncMock, return_value="video_path.mp4")
+    
+    # Mock avatar info loader
+    mocker.patch(
+        "aafactory.chat.interface._load_avatar_infos_for_chat",
+        return_value=(
+            "Bot", "Friendly", "Knows stuff", "avatar_video.mp4",
+            "elevenlabs", "voiceid", "path/to/rec", "transcript", "en", "avatar.png"
+        )
+    )
+
+    # Print initial state
+    print(f"Initial CHAT_HISTORY: {CHAT_HISTORY}")
+
+    avatar_info = _load_avatar_infos_for_chat()
+
+    # Test with elevenlabs
+    mocker.patch("aafactory.chat.interface.send_request_to_elevenlabs", new_callable=mocker.AsyncMock, return_value="audio_path.wav")
+
+    result1 = await send_request_to_llm(
```

Comment:
result1 -> first_llm_answer

Reply:
also I think you could rather focus on the function send_request_to_open_ai. As this is the one consuming the CHAT_HISTORY